xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -1404,10 +1404,10 @@ namespace PVR
       if (channelGroup)
       {
         // try to start playback of first channel in this group
-        std::vector<PVRChannelGroupMember> groupMembers(channelGroup->GetMembers());
+        const std::vector<std::shared_ptr<PVRChannelGroupMember>> groupMembers(channelGroup->GetMembers());
         if (!groupMembers.empty())
         {
-          return SwitchToChannel(CFileItemPtr(new CFileItem((*groupMembers.begin()).channel)), true);
+          return SwitchToChannel(CFileItemPtr(new CFileItem((*groupMembers.begin())->channel)), true);
```

Comment:
while you are at it: `return SwitchToChannel(std::make_shared<CFileItem>(*groupMembers.begin())->channel), true);`